pkg/apis/intelligence/v1alpha1/types.go

```diff
@@ -16,11 +16,14 @@ package v1alpha1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-type JobType string
-
 const (
-	NPRecommendationJobInitial    JobType = "Initial"
-	NPRecommendationJobSubsequent JobType = "Subsequent"
+	NPRecommendationJobInitial    string = "Initial"
+	NPRecommendationJobSubsequent string = "Subsequent"
+	NPRecommendationJobNew        string = "NEW"
+	NPRecommendationJobScheduled  string = "SCHEDULED"
+	NPRecommendationJobRunning    string = "RUNNING"
+	NPRecommendationJobCompleted  string = "COMPLETED"
+	NPRecommendationJobFailed     string = "FAILED"
```

Comment:
Maybe we can remove `Job` in these names? I think `Job` in `NPRecommendationJobInitial` refers to the job type. But in terms of state, it is confused to still have the word `Job` in the name?